src/components/Login/loginStyle.css

```diff
@@ -0,0 +1,231 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
```

Comment:
If this is the font we want to use across the entire site, look into moving this import to the top of the index.css. Then add Open-Sans to the start of the body tag's "font-family" in the index.css file.